src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1283,7 +1285,8 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         ptr_loads = ptr_loads_copy;
         llvm::Value *pos = tmp;
 
-        tmp = list_api->read_item(plist, pos,
+        list_api->enable_bounds_checking = enable_bounds_checking;
```

Comment:
`enable_bounds_checking` should be a static data member of LLVMList. Right now its localised to only one object. So passing it as an argument or doing `list_api->enable_bounds_checking = enable_bounds_checking` is the same thing.

Reply:
In that case, how do I assign the `enable_bounds_checking` value?

Reply:
Remove these assignments from everywhere in this file.